app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -434,6 +440,11 @@ def inspector_proxy_env_variables
                    :name    => PROXY_ENV_VARIABLES).each_with_object([]) do |att, env|
       env << {:name  => att.name.upcase,
               :value => att.value} unless att.value.blank?
-    end
+    end << { :name => INSPECTOR_AUTH_TOKEN,
+             :value => auth_token}
```

Comment:
I don't think it is a very clear way to add those. `inspector_proxy_env_variables` suppose to return environment variables for handling http proxies. Maybe you can add it in https://github.com/ManageIQ/manageiq/pull/14622/files#diff-0a5ccb8a9fe991f71c0d38d1783ac03eR403 directly? I think it will be much clearer.